.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm install
+    - run: npm run asbuild
```

Comment:
Could we check here that rebuilding doesn't change the `build/` directory?

Reply:
Are you sure we want that enforced on pull requests? Typically I would ask external contributors to omit a `build/` directory from their PRs for easier review, and rebuild it myself (e.g. in a release script). Maybe this could be left for a later addition?

Reply:
As we have `build/` directory checked-in, I'd like to have some way to ensure that the `main` branch is always in sync with itself. This doesn't have to happen on pull requests, so let's add this later.